### PR TITLE
Moved Single scraper to own Folder

### PR DESCRIPTION
**Pull request by Kennuel**
I moved the single Scraper to an own folder; such that its easier to create a dockerfile.

### CODE REVIEW

File: singleScraper/dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Start with a base image containing Java runtime
```
Comment (Kennuel) [inline singleScraper/dockerfile]: WOW....

Comment (Kennuel) [reply singleScraper/dockerfile]: Just wrong, nevertheless it seems that its not that easy to run pupperteer inside a docker.

**Comment by MarekWasilewski** [comment]
I would suggest two minor adaptions:

1. Move results/scraped messages into own subfolder
2. Rename package from singleScraper to SingleScraper (root package folder)

**Comment by Kennuel** [comment]
> 1. Move results/scraped messages into own subfolder

Yes, we should do this for now. Later it will be send to kafka as a message and then its saved in a database and not anymore on the filesystem.

> 2. Rename package from singleScraper to SingleScraper (root package folder)

Yes we should do this.

**Pull request merged**